examples/images/diffusion/README.md

```diff
@@ -39,9 +39,17 @@ pip install transformers==4.19.2 diffusers invisible-watermark
 pip install -e .
 ```
 
+### install lightning
+```
+git clone https://github.com/1SAA/lightning.git
+git checkout strategy/colossalai
+export PACKAGE_NAME=pytorch
+pip install .
+```
+
 ### Install [Colossal-AI v0.1.10](https://colossalai.org/download/) From Our Official Website
 ```
-pip install colossalai==0.1.10+torch1.11cu11.3 -f https://release.colossalai.org
+pip install colossalai==0.1.11rc5+torch1.12cu11.3 -f https://release.colossalai.org
```

Comment:
Should it be `0.1.12` now?

Reply:
I want to provide a stable version of colossalai for users, which will not cause conflict

Reply:
0.1.12 is a stable version. you should use the latest release version.

Reply:
ok